src/contexts/DataContext.tsx

```diff
@@ -11,6 +11,20 @@ interface DataContextState {
   setQueryResult: React.Dispatch<React.SetStateAction<QueryResult[]>>;
 }
 
+// Utility function to map the environment variable to the SearchSource enum
+export const getDefaultSearchSource = (): SearchSource => {
```

Comment:
I'm having trouble believing this is the simplest code for this when this works:

```typescript
const enum EDirection {
  Up,
  Down,
  Left,
  Right,
}

const lookup = EDirection["Down"]

console.log(lookup) // outputs 1
```

Reply:
Lint does not allow the notation `EDirection["Up"]`. Says `["Up"] is better written in dot notation` meaning something like `EDirection.Up`. It pained me to commit it too. I searched for a better way with no avail. I didn't see the `EDirection["Up"]` notation before though.

Reply:
Done. Strangely, `EDirection["Down"]` doesn't pass Lint. Says `["Down"] is better written in dot notation`. But `EDirection["Down" as keyof typeof Edirection]` passes.